packages/web3-react/src/components/AlephiumConnect.tsx

```diff
@@ -86,17 +80,38 @@ export const AlephiumConnectProvider: React.FC<AlephiumConnectProviderProps> = (
     customTheme,
     setCustomTheme,
     addressGroup,
-    keyType,
+    keyType: keyType ?? 'default',
     // Other configuration
     errorMessage
   }
 
   return (
-    <AlephiumConnectContext.Provider value={value}>
+    <ConnectSettingContext.Provider value={value}>
       <ThemeProvider theme={defaultTheme}>
         {children}
         <AlephiumConnectModal theme={theme} mode={mode} customTheme={customTheme} />
       </ThemeProvider>
-    </AlephiumConnectContext.Provider>
+    </ConnectSettingContext.Provider>
   )
 }
+
+export const AlephiumConnectProvider: React.FC<{ children?: React.ReactNode }> = ({ children }) => {
```

Comment:
IIUC, now we need to wrap the app with both `AlephiumConnectProvider` and `ConnectSettingProvider`. Maybe we should update [README](https://github.com/alephium/alephium-web3/blob/master/packages/web3-react/README.md?plain=1#L38). I will update docs as well.

Reply:
Sorry, just saw in the next commit we have included `ConnectSettingProvider` in `AlephiumConnectProvider`.

Reply:
Will update the readme, thanks!